rest/taskana-rest-spring/src/test/java/pro/taskana/user/rest/UserControllerIntTest.java

```diff
@@ -0,0 +1,145 @@
+package pro.taskana.user.rest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static pro.taskana.common.test.rest.RestHelper.TEMPLATE;
+
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.HttpStatusCodeException;
+
+import pro.taskana.common.rest.RestEndpoints;
+import pro.taskana.common.test.rest.RestHelper;
+import pro.taskana.common.test.rest.TaskanaSpringBootTest;
+import pro.taskana.user.rest.models.UserRepresentationModel;
+
+/** Tests the endpoints of the UserController. */
+@TaskanaSpringBootTest
+class UserControllerIntTest {
```

Comment:
Just a FYI: If the REST doc test already cover the operations, no extra int tests are needed...since we only have CRUD here, the RESt doc tests would have been enough. But don't delete these if you already wrote them :)

Reply:
Ohh, nice to know 😂